source/chapters/library.rst

```diff
@@ -866,6 +867,86 @@ different location. Select :guilabel:`Use Default Library` to reset.
+|            | ID3        | FLAC      | M4A       | Ogg Vorbis    | Description               |
++============+============+===========+===========+===============+===========================+
+| Analysis   | No         | No        | No        | No            | Serato Analysis version   |
++------------+------------+-----------+-----------+---------------+---------------------------+
+| Autotags   | No         | No        | No        | No            | BPM and Gain values       |
++------------+------------+-----------+-----------+---------------+---------------------------+
+| Beatgrid   | Yes        | Yes       | Yes       | No            | Beatgrid Markers          |
++------------+------------+-----------+-----------+---------------+---------------------------+
+| Markers    | Yes        | N/A       | Yes       | No            | Hotcues, Saved Loops, etc |
++------------+------------+-----------+-----------+---------------+---------------------------+
+| Markers2   | Yes        | Yes       | Yes       | ?             | Hotcues, Saved Loops, etc |
++------------+------------+-----------+-----------+---------------+---------------------------+
+| Offsets    | No         | No        | No        | No            |                           |
++------------+------------+-----------+-----------+---------------+---------------------------+
+| Overview   | No         | No        | No        | No            | Overview Waveform data    | 
++------------+------------+-----------+-----------+---------------+---------------------------+
+| RelVol     | N/A        | No        | No        | N/A           |                           |
++------------+------------+-----------+-----------+---------------+---------------------------+
+| VideoAssoc | N/A        | No        | No        | N/A           |                           |
++------------+------------+-----------+-----------+---------------+---------------------------+
+
+
+Import Loops from Serato
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+From Mixxx 2.3, saved loops from Serato can be imported, but only the first loop is usable in Mixxx 2.3. 
+Proper support for importing and restoring saved loops from Serato is scheduled for Mixxx 2.4. 
+
+For now, we just import these loops into the Mixxx database and allow using their start position as regular hotcues. 
+With this in place, you can start using your loops once Mixxx 2.4 is out and avoid reimporting track meta data.
+
+Imports not currently supported
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Support for importing track overview waveform image stored in Serato into Mixxx and smart crates has not been provided in Mixxx 2.3. 
+Serato flip macros cannot be imported either because Mixxx does not offer that kind of functionality yet.
+
+Currently, you cannot write cues from Mixxx into Serato metadata tags. 
+We are working hard to implemment support for exporting Serato metadata tags in upcoming Mixxx versions, stay tunned.
```

Comment:
With the changes I suggested above this can be removed.

```suggestion
```

Reply:
Thank you so much @Holzhaus. I would implement this as soon as possible

Reply:
> With the changes I suggested above this can be removed.

Ok! I would remove the table and effect other changes you requested for

Reply:
> With the changes I suggested above this can be removed.

Hello @Holzhaus I just implemented the changes, Please review!